src/assets/scripts/client/utilities/validatorUtilities.js

```diff
@@ -0,0 +1,36 @@
+import _isArray from 'lodash/isArray';
+import _isEmpty from 'lodash/isEmpty';
+import _isObject from 'lodash/isObject';
+
+/**
+ * This will return true if it is not an object or it is empty
+ *
+ * @funtion isNotObjectOrIsEmpty
+ * @param input {object}
+ * @return {boolean}
+ */
+export const isObjectEmpty = (value) => {
```

Comment:
this should be renamed to `isEmptyObject`.  be sure to update the methods below with this name, too.